packages/react/src/themes/teams/components/Alert/alertStyles.ts

```diff
@@ -83,12 +101,42 @@ const alertStyles: ComponentSlotStylesInput<AlertProps, AlertVariables> = {
     flexGrow: 1,
   }),
 
-  action: ({ variables: v }): ICSSInJSStyle => ({
+  action: ({ variables: v, props }): ICSSInJSStyle => ({
     height: v.actionSize,
     minWidth: v.actionSize,
     margin: `-${v.borderWidth} 0`,
     color: v.actionColor || 'currentColor',
+    border: `${pxToRem(1)} solid transparent`,
+    borderRadius: v.borderRadius,
+
     ':focus': { outline: 0 },
+
+    ':hover': {
+      color: 'currentColor',
+
+      [`& .${teamsIconClassNames.filled}`]: {
+        display: 'block',
+      },
+
+      [`& .${teamsIconClassNames.outline}`]: {
+        display: 'none',
+      },
+    },
+
+    ...(props.isFromKeyboard && {
+      ':focus': {
+        borderColor: v.focusInnerBorderColor,
+        boxShadow: `0 0 0 ${pxToRem(1)} ${v.focusOuterBorderColor}`,
```

Comment:
shouldn't boxShadow be a design term, hence part of variables or siteVariables?

Reply:
Yes, but I will remove these styles for now until I can test the focus styles with `isFromKeyboard` working

Reply:
makes sense, I'll update you once I have a PR